src/pyscipopt/scip.pyx

```diff
@@ -203,6 +203,13 @@ cdef class PY_SCIP_BRANCHDIR:
     FIXED     = SCIP_BRANCHDIR_FIXED
     AUTO      = SCIP_BRANCHDIR_AUTO
 
+cdef class PY_SCIP_BENDERSENFOTYPE:
```

Comment:
Please add this new class to `__init__.py`, so we can import it like this: `SCIP_BENDERSENFOTYPE`.